core/util/src/main/scala/net/liftweb/util/HttpHelpers.scala

```diff
@@ -138,7 +138,7 @@ trait HttpHelpers {
   */
   def toHashMap[A,B](in : Map[A,B]) : HashMap[A,B] = {
     val ret = new HashMap[A,B];
-    in.keysIterator.foreach { k => ret += Pair(k, in(k)) }
+    in.keysIterator.foreach { k => ret += ((k, in(k))) }
```

Comment:
Yeah sorry was thinking of `->` rather than `,` in the middle; I *think* that fixes the precedence issues that cause the error you're seeing. Could be wrong though.